src/components/molecules/TableComponent/TableComponent.tsx

```diff
@@ -75,9 +75,8 @@ const TableComponent: React.FunctionComponent<TableComponentPropsType> = ({
     isJazzBar
       ? {}
       : {
-          height: `${table.rows && table.rows * 50 + 65}px`,
```

Comment:
Removed this. There's a min-height on the component already and it doesn't need anything further. It grows as it should.